test/test_transcribe_utterance_record.py

```diff
@@ -0,0 +1,53 @@
+"""
+    FreeClimb API
+
+    FreeClimb is a cloud-based application programming interface (API) that puts the power of the Vail platform in your hands. FreeClimb simplifies the process of creating applications that can use a full range of telephony features without requiring specialized or on-site telephony equipment. Using the FreeClimb REST API to write applications is easy! You have the option to use the language of your choice or hit the API directly. Your application can execute a command by issuing a RESTful request to the FreeClimb API. The base URL to send HTTP requests to the FreeClimb REST API is: /apiserver. FreeClimb authenticates and processes your request.  # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Contact: support@freeclimb.com
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import sys
+import unittest
+from datetime import datetime, date
+import pytest
+
+import freeclimb
+
+from freeclimb.model.transcribe_utterance_record import TranscribeUtteranceRecord  # noqa: E501
+
+class TestTranscribeUtteranceRecord(unittest.TestCase):
+    """TranscribeUtteranceRecord unit test stubs"""
+
+    def setUp(self):
+        self.model = TranscribeUtteranceRecord()
+    
+    def test_save_recording(self):
+        """Test TranscribeUtteranceRecord.save_recording"""
+        self.model.save_recording = False
+        assert self.model.get("save_recording") == False
+        
+         
```

Comment:
Can we adjusts the formatting here? Looks like the `test_max_length_sec` and `test_rcrd_termination_silence_time_ms` tests are broken up a bit.